health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -1530,4 +1530,108 @@
+
+      const requestDate = data?.body?.inbox?.moduleSearchCriteria?.range?.requestDate;
+
+      if (requestDate?.startDate && requestDate?.endDate) {
+        const fromDate = new Date(requestDate.startDate).getTime();
+        const toDate = new Date(requestDate.endDate).getTime();
+
+        data.body.inbox.moduleSearchCriteria.fromDate = fromDate;
+        data.body.inbox.moduleSearchCriteria.toDate = toDate;
+      }
+      else{
+        delete data.body.inbox.moduleSearchCriteria.fromDate;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)